backends-clickhouse/src/test/scala/io/glutenproject/execution/extension/CustomAggExpressionTransformer.scala

```diff
@@ -61,4 +62,22 @@ case class CustomAggExpressionTransformer() extends ExpressionExtensionTrait {
         }
     }
   }
+
+  /** Get the custom agg function substrait name and the input types of the child */
+  override def buildCustomAggregateFunction(
+      aggregateFunc: AggregateFunction): (Option[String], Seq[DataType]) = {
+    val substraitAggFuncName = aggregateFunc match {
+      case customSum: CustomSum =>
+        if (customSum.dataType.isInstanceOf[LongType]) {
+          Some("custom_sum")
+        } else {
+          Some("custom_sum_double")
+        }
+      case _ =>
+        throw new UnsupportedOperationException(
```

Comment:
Return `None` if we check the returned Option on calling side, as mentioned in other comment.